src/pages/home/report/ReportActionItemMessageEdit.js

```diff
@@ -53,6 +54,9 @@ const propTypes = {
     /** Whether or not the emoji picker is disabled */
     shouldDisableEmojiPicker: PropTypes.bool,
 
+    /** Stores user's preferred skin tone */
+    preferredSkinTone: PropTypes.oneOfType([PropTypes.number, PropTypes.string]).isRequired,
```

Comment:
I don't think this is required. Do same as on `ReportActionItem` not required and set default to `-1`.

Reply:
👍🏻